doc/uml/class_m2gui.uml

```diff
@@ -26,6 +26,7 @@ MainWindow ..> QMessageBoxAsync
 Model *-- FaultManager
 Model *-- UtilityMonitor
 Model *-- SignalControl : emit()
+Model *-- SignalPowerSystem : emit()
```

Comment:
This is misleading. You aren't emitting a class, you are emitting its member, is_state_updated. But also that design makes your code much harder to read and understand. Let's discuss that in person once I am back.

Reply:
I think this is what I could do for the class UML to explain the relationship among classes/signals. The documents and class diagrams are provided to let the developers know the code architecture. Thanks!